pytest.ini

```diff
@@ -0,0 +1,3 @@
+[pytest]
+DJANGO_SETTINGS_MODULE = ultimatejobweb.settings
```

Comment:
Setting is located under ultimatejob.setting, did you try running the tests?

Reply:
If you use this settings, it is for pipenv install pipenv install --dev pytest-django and not pytest, compare your Pipfile